### ISSUE TITLE
No connection with arduino uno

### ISSUE THREAD

**Issue by hugovw1976** [speeduino/Ardu-Stim#9]
Hi I try to use this version of ardustim but can connect with arduino uno , I can upload ardustim.ino to arduino uno or arduino mega via arduino ide, but when open Ardu_Stim.0.04
can connect with arduino uno just arduino mega, I only have one arduino mega an is for speeduino I need help. thanks

**Comment by mike501** [comment]
Doesn't help directly but I can recompile ardustim for uno and have it happily running. When you say you can't connect do you mean the UI? Have you rescanned the ports between using the mega and the uno after the compile has said its finished uploading?

**Comment by JTR57** [comment]
Same here but with a Nano. Can't flash the firmware via Ardu_Stim.0.0.4 and 0.0.3 doesn't connect to the old firmware. The old version of ardustim that uses droid4arduino is working properly.

**Comment by kinglevel** [comment]
Same here, had to compile it manually in arduino IDE, and now it connected. 
UNO

**Comment by m1t1k1** [comment]
Hello, and can it be done to connect to other versions of hardware such as WAVGAT UNO R3

**Comment by AmesisProject** [comment]
Yes the version of this git no longer works with the .ino uploaded and the index.html UI. it uses the software in .exe version which is in -> "code" and on the right in "Releases", here you will find the .exe and when you open it you will see your arduino appear and it is from this interface that you will upload the code then you click on connect and everything works.
https://github.com/speeduino/Ardu-Stim/releases
